versions/run.jet.ver

```diff
@@ -5,3 +5,5 @@ export hpss_ver=
 export ncl_ver=6.6.2
 export R_ver=4.0.2
 export gempak_ver=7.4.2
+
+source "${HOMEgfs:-}/versions/run.spack.ver"
```

Comment:
It's too far away to suggest the change, but the same goes for L1-2 here and the other RDHPCS run ver files.

Reply:
The same goes for `run.spack.ver`.  The compiler versions should be removed from there rather than the machine files.

Reply:
There are no compiler and/or mpi versions in `run.spack.ver`.